### PR TITLE
Docs: adding guidance examples

### PR DESCRIPTION
**Pull request by lukeconibear**
### Summary
Adding examples to the guidance documents.

### Changes
- [x] `book/guidance/data_management.md`: Adding examples.
- [x] `book/guidance/software_management.md`: Adding example.

### CODE REVIEW

File: book/guidance/data_management.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+- [Research data management timeline (Library)](https://library.leeds.ac.uk/info/14062/research_data_management/143/research_data_management_timeline)
+- [Research data management policy (Library)](https://library.leeds.ac.uk/info/14062/research_data_management/68/research_data_management_policy)
+- [Data management plan (The Turing Way)](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-dmp.html)
+
+#### What classification is the data?
+
+- [Classification categories (Data Protection)](https://dataprotection.leeds.ac.uk/managing-electronic-data/#classifying-data)
+- [LASER (Leeds Analytic Secure Environment for Research): Data Risk Tiering](https://lida-data-analytics-team.github.io/laserdocs/docs/laser_info/tiering.html)
+- [Information Management Guide (Data Protection)](https://dataprotection.leeds.ac.uk/wp-content/uploads/sites/48/2019/05/Information-Management-Guide.pdf)
+- [Information Protection Policy (Data Protection)](https://leeds.service-now.com/it?id=kb_article&sys_id=6038bfbc0fae728089d7f55be1050e9d)
```
Comment (jhodrien) [inline book/guidance/data_management.md:30]:
```suggestion
- [Information Protection Policy (Data Protection)](https://it.leeds.ac.uk/it?id=kb_article&sys_id=6038bfbc0fae728089d7f55be1050e9d)
```

File: book/guidance/recommended_platforms.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  - [ARC4](https://arcdocs.leeds.ac.uk/systems/arc4.html#standard-nodes)
+    - 149 standard nodes, each with 40 cores
+    - 2 high-memory nodes, each with 40 cores
+- Cloud computing
+  - [Microsoft Azure - Compute optimised](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-compute)
+
+### Do you need to use GPUs?
+
+- Buy a local laptop / workstation with a GPU
+  - [Purchase form](https://leeds.service-now.com/it?id=sc_cat_item&sys_id=a649379c0f2f9b40a82247ece1050e25)
```
Comment (jhodrien) [inline book/guidance/recommended_platforms.md:54]:
```suggestion
  - [Purchase form](https://it.leeds.ac.uk/it?id=sc_cat_item&sys_id=a649379c0f2f9b40a82247ece1050e25)
```

File: book/guidance/recommended_platforms.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    - 2 x [general purpose GPU](https://en.wikipedia.org/wiki/General-purpose_computing_on_graphics_processing_units) nodes, each with 2 x [NVIDIA Tesla K80](https://www.nvidia.com/en-gb/data-center/tesla-k80/)
+    - 6 x [general purpose GPU](https://en.wikipedia.org/wiki/General-purpose_computing_on_graphics_processing_units) nodes, each with 4 x [NVIDIA Tesla P100](https://www.nvidia.com/en-gb/data-center/tesla-p100/)
+  - [ARC4](https://arcdocs.leeds.ac.uk/systems/arc4.html#gpgpu-nodes)
+    - 3 x [general purpose GPU](https://en.wikipedia.org/wiki/General-purpose_computing_on_graphics_processing_units) nodes, each with 4 x [NVIDIA Tesla V100](https://www.nvidia.com/en-gb/data-center/tesla-v100/)
+  - [Bede](https://bede-documentation.readthedocs.io/en/latest/hardware/index.html)
+    - 2 x login GPU nodes each with 4 x [NVIDIA Tesla V100](https://www.nvidia.com/en-gb/data-center/tesla-v100/)
+    - 32 x GPU nodes each with 4 x [NVIDIA Tesla V100](https://www.nvidia.com/en-gb/data-center/tesla-v100/)
+    - 4 x "inference" GPU nodes each with 4 x [NVIDIA Tesla T4](https://www.nvidia.com/en-gb/data-center/tesla-t4/)
+  - [JADE-2](http://docs.jade.ac.uk/en/latest/index.html)
```
Comment (jhodrien) [inline book/guidance/recommended_platforms.md:65]:
```suggestion
  - [JADE-2](https://docs.jade.ac.uk/en/latest/)
```

File: book/guidance/software_management.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  - [Version control (The Alan Turing Institute)](https://alan-turing-institute.github.io/rse-course/html/module04_version_control_with_git/04_00_introduction.html)
+
+#### [Reproducibility](https://the-turing-way.netlify.app/reproducible-research/reproducible-research.html)
+
+##### Package management
+
+- [Virtual environments](https://carpentries-incubator.github.io/python-intermediate-development/12-virtual-environments/index.html)
+  - e.g., [conda](https://docs.conda.io/projects/conda/en/latest/index.html), [pip](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#installing-packages-using-pip-and-virtual-environments) and [venv](https://docs.python.org/3/library/venv.html) (Python), [virtualenv](https://virtualenv.pypa.io/en/latest/) (Python), [pipenv](https://pipenv.pypa.io/en/latest/) (Python), [poetry](https://python-poetry.org/) (Python), [spack](https://spack.readthedocs.io/en/latest/) (High Performance Computers), [binder](https://mybinder.readthedocs.io/en/latest/index.html) ([guide](https://the-turing-way.netlify.app/reproducible-research/renv/renv-binder.html)), [conda-store](https://conda-store.readthedocs.io/en/latest/)
```
Comment (jhodrien) [inline book/guidance/software_management.md:41]: I question labelling here.  Conda I think extends well beyond Python, and I'm not sure labelling spack as an HPC tool (even though that's where its origins lie) is necessarily helpful.  How to phrase and indeed group.  Would it be better grouping the the Python specific tools, then listing conda and spack as sort of "user level package management systems".  I don't like that term, but I feel that's more what we need to angle for.

File: book/guidance/recommended_platforms.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    - 63 x [DGX-MAX-Q](https://www.nvidia.com/en-gb/data-center/dgx-systems/dgx-1/) Nodes, each with 8 x [NVIDIA Tesla V100](https://www.nvidia.com/en-gb/data-center/tesla-v100/)
+- Cloud computing
+  - [Microsoft Azure - GPU optimised](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-gpu)
+
+## Cost
+
+- *Azure*  
+  - When considering costs relating to research platforms it's important to think of cloud resources differently to on-campus hardware.  
+  - Cloud resources are typically not designed to be on all the time and all aspects of a cloud computational setup have costs associated i.e., you'll pay for your virtual machine (VM) whilst it's on, but you'll also need to pay for it's storage, virtual network, and associated public IP address.
```
Comment (jhodrien) [inline book/guidance/recommended_platforms.md:74]:
```suggestion
  - Cloud resources are typically not designed to be on all the time and all aspects of a cloud computational setup have costs associated i.e., you'll pay for your virtual machine (VM) whilst it's on, but you'll also need to pay for its storage, virtual network, and associated public IP address.  
```

File: book/guidance/data_management.md (hunk 1 above)
Comment (lukeconibear) [reply book/guidance/data_management.md:30]: Added over to #100

File: book/guidance/recommended_platforms.md (hunk 2 above)
Comment (lukeconibear) [reply book/guidance/recommended_platforms.md:54]: Added over to #100

File: book/guidance/recommended_platforms.md (hunk 3 above)
Comment (lukeconibear) [reply book/guidance/recommended_platforms.md:65]: Added over to #100

File: book/guidance/software_management.md (hunk 4 above)
Comment (lukeconibear) [reply book/guidance/software_management.md:41]: Added over to #100

File: book/guidance/recommended_platforms.md (hunk 5 above)
Comment (lukeconibear) [reply book/guidance/recommended_platforms.md:74]: Added over to #100

File: book/guidance/data_management.md [hunk 6]
```diff
@@ -152,3 +152,9 @@ This guide asks a range of questions to consider when planning your data managem
 - [Individual Rights (Data Protection)](https://dataprotection.leeds.ac.uk/individual-rights/)
 - [Reporting a Data Security Incident (Data Protection)](https://dataprotection.leeds.ac.uk/reporting-a-data-security-incident/)
 - [Further information (The Turing Way)](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-dmp.html#further-reading-recommendations)
+
+## Examples
+
+- [DMPOnline](https://dmponline.dcc.ac.uk/public_plans)
+- [Published DMPs](https://riojournal.com/browse_journal_articles.php?section_type%5B%5D=231)
+- [Digital Curation Centre](http://www.dcc.ac.uk/resources/data-management-plans/guidance-examples)
```
Comment (jhodrien) [inline book/guidance/data_management.md:160]:
```suggestion
- [Digital Curation Centre](https://www.dcc.ac.uk/resources/data-management-plans/guidance-examples)
```

**Review by jhodrien — APPROVED** [review]
Looks like an interesting set of examples to me.

**Pull request merged**